### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by caballerofelipe** [OpenBB-finance/OpenBB#6076] [Ideas]
Is there somewhere where the roadmap to OpenBB is published?

I've been using OpenBB for not so long but for instance it feels OpenBB Terminal isn't going to be supported for much longer as you can see this in the Pypi instalation doc:

<img width="822" alt="Screenshot 2024-02-14 at 11 32 57" src="https://github.com/OpenBB-finance/OpenBBTerminal/assets/11650766/4d009f0f-64e1-407a-b035-ff7d39094515">

I want to learn how to use OpenBB, I really like using the terminal and Jupyter so it would be cool to understand where the project is going and what tools will have priority over others. I'm thinking more about a more general view on the future of the project not necessarily the planning of things like a specific feature or functionality, I guess I'm thinking more about a philosophical view on where the project is going...

**Comment by tilakpatel22** [comment]
@caballerofelipe I have been building a similar financial terminal and I would love if you check and provide me feedbacks on it - https://github.com/Fincept-Corporation/FinceptTerminal
